app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if type(other) == int or type(other) == float:
```

Comment:
You should extract type checking into a separate methods that returns int | float, because right now your code violates DRY principle.
Also, it's easier to check for `isinstance(obj, Distance)` instead of int/float.